src/lib/system/repositories/settings_repository_impl.dart

```diff
@@ -14,6 +14,7 @@ class SettingsRepositoryImpl implements SettingsRepository {
 
   final BehaviorSubject<bool> _manageExternalStorageGrantedSubject = BehaviorSubject();
   final BehaviorSubject<String> _fileExtensionsSubject = BehaviorSubject();
+  final BehaviorSubject<bool> _playAlbumsInOrderSubject = BehaviorSubject();
```

Comment:
You need to add a listener to the stream of the data source. Same as in line 10:

    _settingsDataSource.playAlbumsInOrderStream.listen(_playAlbumsInOrderSubject.add);

The rationale is that this way, we transform the normal stream into a `ValueStream` that has (IMO) some beneficial properties for the usage in the UI. This setup might be a bit convoluted, but so far, it worked well for me and I keep to this recipe.